### ISSUE TITLE
I have some questions when I tryed to train on your code

### ISSUE THREAD

**Issue by young169** [1M50RRY/dfdc-kaggle-solution#1]
Congratulations and thanks for sharing your code. However, I'd like to know what the 'metadata.csv' is made of, for the original given data is a json file.Is is made of a 'data' field which is the path to video and 'label' field either 0 or 1? And what does the raw_models and meta_models in weigths_cfg.py mean? Are they all your private training results or some pre-trained models for public? I'm just a beginner and eager for your answers :)

**Comment by 1M50RRY** [comment]
Metadata.csv is a csv representation of json file, you are right, it is just video path and 0-1 label. About weights, I forgot to upload it, so I did it now, check inference/pretrained.

**Comment by young169** [comment]
And how to make train and val data? Can you show me an example of  'metadata.csv' and 'data/train' and 'data/img_val'? According to my understanding, I just put the face images under data/train (or data/img_val) and metadata.csv records its path and label?

**Comment by 1M50RRY** [comment]
metadata.csv contains filename or path and label (0 - real, 1 - fake), data/train contains faces from 0-39 data chunks, data/img_val contains faces from 40-49 data chunks

**Comment by young169** [comment]
Hi, sorry to bother you further, but I cannot find the file of your meta_models in the code : 'pretrained/MetaModel 0.264787397152165.pth', and I found a new model 'MetaModel 0.256337642808031.pth' according to your uploads, so I replaced the latter one in the code. And I got an error when training stacked models:'size mismatch for fc.weight: copying a param with shape torch.Size([1, 4]) from checkpoint, the shape in current model is torch.Size([1, 14]).'

**Comment by 1M50RRY** [comment]
Uploaded MetaModel 0.264787397152165, but I didn't use it in inference.
